setup.cfg

```diff
@@ -12,7 +12,32 @@ tag_prefix =
 parentdir_prefix = great_expectations-
 
 [flake8]
-max-line-length = 200
-exclude = tests/*,docs/*
+exclude = tests/*,
+    docs/*,
+    build/*,
+    assets/*,
+    contrib/*,
+    versioneer*,
+    examples/*,
+    # TODO: remove the items below and fix linting issues
```

Comment:
Can you file a ticket, tag it with `hygiene` and put it in the `Mario QOL` epic?

Reply:
https://superconductive.atlassian.net/browse/GREAT-1146